.dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.10.0
+
+LABEL maintainer="Gorgeous-Ghouls"
+
+RUN pip install poetry
```

```suggestion
RUN apt-get -y update
RUN apt-get -y install git curl
RUN curl -sSL https://install.python-poetry.org | python3 -
```
as installing with pip is deprecated